libs/interpreter-lib/src/interpreter.ts

```diff
@@ -82,16 +83,25 @@ export async function interpretModel(
             ', ',
           )}.`,
       );
-    process.exit(ExitCode.FAILURE);
+    return ExitCode.FAILURE;
```

Comment:
I wonder if we should distinguish between different kinds of `FAILURES`, like `CONFIGURATION_FAILURE`, `RUNTIME_FAILURE`, `RUNTIME_ERROR`, and so on.

Reply:
Would be good, yeah. Note though, this is the exit code of the process so we can not freely define new ones (afaik). We definitely should have some form of more elaborate error output outside of the exit code though, like writing a logfile.